<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThankyouWidget</class>
 <widget class="QWidget" name="ThankyouWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1097</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>225</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="2">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>22</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>22</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>226</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>610</width>
       <height>480</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>610</width>
       <height>480</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(57, 91, 129);
border-radius:8px;</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lblThanku">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background:transparent;
border-image:transparent;
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>THANKS FOR YOUR VISIT</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblName">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background:transparent;
border-image:transparent;
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>NAME</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_5">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>25</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QWidget" name="printerForm" native="true">
          <property name="minimumSize">
           <size>
            <width>341</width>
            <height>211</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>341</width>
            <height>211</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-image: url(:/images/Images/print_bk.png);</string>
          </property>
          <widget class="QLabel" name="lbl_name">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>50</y>
             <width>190</width>
             <height>18</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">image: url(:/images/Images/blank.png);
background:transparent;
color: rgb(47, 47, 47);</string>
           </property>
           <property name="text">
            <string>SORNA DEVADAS</string>
           </property>
          </widget>
          <widget class="QLabel" name="lbl_company">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>80</y>
             <width>190</width>
             <height>22</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>8</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">image: url(:/images/Images/blank.png);
background:transparent;
color: rgb(47, 47, 47);</string>
           </property>
           <property name="text">
            <string>ANROS GROUP</string>
           </property>
          </widget>
          <widget class="QLabel" name="lbl_person_visiting">
           <property name="geometry">
            <rect>
             <x>69</x>
             <y>110</y>
             <width>150</width>
             <height>20</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">image: url(:/images/Images/blank.png);
background:transparent;
color: rgb(47, 47, 47);</string>
           </property>
           <property name="text">
            <string>William Borg</string>
           </property>
          </widget>
          <widget class="QLabel" name="lbl_date">
           <property name="geometry">
            <rect>
             <x>70</x>
             <y>160</y>
             <width>131</width>
             <height>20</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">image: url(:/images/Images/blank.png);
background:transparent;</string>
           </property>
           <property name="text">
            <string>00-00-0000</string>
           </property>
          </widget>
          <widget class="QLabel" name="lbl_photo">
           <property name="geometry">
            <rect>
             <x>210</x>
             <y>60</y>
             <width>116</width>
             <height>116</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>116</width>
             <height>116</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>116</width>
             <height>116</height>
            </size>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 0);
border-radius: 4px;
border-width:1px;
border:solid;</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="lineWidth">
            <number>0</number>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
          <widget class="QLabel" name="lbl_date_2">
           <property name="geometry">
            <rect>
             <x>70</x>
             <y>140</y>
             <width>131</width>
             <height>20</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">image: url(:/images/Images/blank.png);
background:transparent;</string>
           </property>
           <property name="text">
            <string>00:00pm</string>
           </property>
          </widget>
          <widget class="QLabel" name="lbl_type">
           <property name="geometry">
            <rect>
             <x>80</x>
             <y>10</y>
             <width>190</width>
             <height>18</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">image: url(:/images/Images/blank.png);
background:transparent;
color: rgb(47, 47, 47);</string>
           </property>
           <property name="text">
            <string>SORNA DEVADAS</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
          <widget class="QLabel" name="lbl_client">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>190</y>
             <width>301</width>
             <height>16</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:rgb(30,30,30);</string>
           </property>
           <property name="text">
            <string>[Comapny Name]</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>120</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(138, 159, 178);</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:rgb(255,255,255);</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Please remove your sticker and,&lt;/p&gt;&lt;p&gt;place in the bin provided &lt;/p&gt;&lt;p&gt;Thank You&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0">
     <item>
      <widget class="QPushButton" name="btnCheckOut">
       <property name="minimumSize">
        <size>
         <width>420</width>
         <height>36</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>420</width>
         <height>36</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(80, 99, 132);
border-style: solid;
border-color: rgb(18, 67,125);
border-radius: 4px;
border-width: 1px;
color:rgb(255,255,255);</string>
       </property>
       <property name="text">
        <string>YOU HAVE SIGNED OUT</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer_6">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
